### ISSUE TITLE
Repair CI test dependencies for "Check GraphQL Schema" and "snarkyjs integration" tests

### ISSUE THREAD

**Issue by stevenplatt** [MinaProtocol/mina#12458]
At this time, the `Check GraphQL Schema` and `snarkyjs integration` tests are stuck in a queued state due to unmet dependencies in the I pipeline. 

For the `Check GraphQL Schema` test:
```
Agent Targeting Rules	
...

Step Key	
_CheckGraphQLSchema-check-graphql-schema

Step Dependencies	
_MinaArtifactStretch-build-deb-pkg
```

For the `snarkyjs integration`  test:
```
Agent Targeting Rules	
...

Step Key	
_TestnetIntegrationTests-integration-test-snarkyjs

Step Dependencies	
_TestnetIntegrationTests-build-test-executive
_MinaArtifactBuster-archive-buster-docker-image
_TestnetIntegrationTests-build-js-tests
_MinaArtifactBuster-daemon-devnet-buster-docker-image
```

Both tests will need to be updated to move to newer version of deb and docker build to unblock them.

**Comment by stevenplatt** [comment]
This item has been completed and merged into `develop`: https://github.com/MinaProtocol/mina/pull/12470
